demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1069,6 +1073,71 @@ export class DemoMeetingApp
       });
     });
 
+    const languageIdentificationCb = document.getElementById('identify-language-checkbox') as HTMLInputElement;
+    languageIdentificationCb.addEventListener('click', () => {
+      (document.getElementById('language-options').classList.toggle('hidden', !languageIdentificationCb.checked));
+      (document.getElementById('preferred-language').classList.toggle('hidden', !languageIdentificationCb.checked));
+      (document.getElementById('transcribe-language')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('content-identification-checkbox')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('content-redaction-checkbox')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('custom-language-model-checkbox')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('transcribe-entity')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('language-model-input-text')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
```

Comment:
Just checking -- have we confirmed that none of these features work with language ID?

Reply:
As per [transcribe's documentation](https://docs.aws.amazon.com/transcribe/latest/dg/lang-id.html), yes, these features don't. I can confirm by enabling them and sending request to Transcribe.

Reply:
Verified the features do not work with Transcribe. Refer Chime-46836 for testing and verification.